### DISCUSSION TITLE
Replace supbase name on signin page

### DISCUSSION THREAD

**Discussion by rahilnazir-agentic** [supabase/supabase#29743] [Questions]
![Screenshot 2024-10-08 at 11 46 50](https://github.com/user-attachments/assets/1360b789-add3-4d24-a8f5-0ecd5751c1e2)

How can I update the malicious looking supabase user to my project name? See the screenshot

**Comment by GaryAustin1** [comment]
Normally you need to be on the Pro plan and have a custom domain option.

At one time there was a somewhat painful workaround involving contacting Google... https://github.com/orgs/supabase/discussions/2925  Don't know if that still applies or not.
